cms/djangoapps/contentstore/views/course.py

```diff
@@ -768,15 +785,31 @@ def settings_handler(request, course_key_string):
                 'about_page_editable': about_page_editable,
                 'short_description_editable': short_description_editable,
                 'upload_asset_url': upload_asset_url
-            })
+            }
+            if prerequisite_course_enabled:
+                courses, in_process_course_actions = get_courses_accessible_to_user(request)
+                # exclude current course from the list of available courses
+                courses = [course for course in courses if course.id != course_key]
+                if courses:
+                    courses = _remove_in_process_courses(courses, in_process_course_actions)
+                settings_context.update({'pre_requisite_courses': courses})
```

Comment:
Nit: these aren't the pre_requisite courses for this course. They are the list of all possible pre_requisite courses. Naming the variable that way would make the code easier to understand.